api/render.go

```diff
@@ -76,6 +80,9 @@ func (api *RendererAPI) renderTable(w http.ResponseWriter, r *http.Request) {
 		setErrorCode(ctx, w, err)
 		return
 	}
+
+	defer span.End()
```

Comment:
It would be best practice to include the `defer` immediately after the `span` is instantiated. This serves two purposes:
1. Makes the code more maintainable as it creates a pattern that every time a span is created we should see an associated `defer` call.
2. Ensures that the should future spans be added within the current span they will be added to the defer stack in the correct order. The defer will execute in a first in last out order.